content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: bt_wiki
+title: Deployments View
+category: Widgets
+draft: false
+---
+
+Presents information about deployments in a single view. Contains:
+
+- a table showing the deployments in the system (the left side) The deployments
+  can be further filtered.
+  See [Filtering deployments]({{< relref "#filtering-deployments" >}}).
+- a pane with the deployment details divided into tabs (the right side)
+- a map showing the deployments locations
+
+<!-- TODO: screenshot -->
+
+## Table
+
+The table shows deployments that match the current filter.
+See [Filtering deployments]({{< relref "#filtering-deployments" >}})
+for more details on filtering.
+
+The columns shown in the table can be shown and hidden using the widget's
+configuration.
+
+<!--  TODO: screenshot -->
+
+The columns can show:
+
+1. The deployment status represented as an icon
+
+   - No icon - _good_
+   - Yellow spinner - _in progress_
+   - Red exclamation mark - _requires attention_
+
+   See [the Environments and Services section]({{< relref "#environments-and-services" >}})
+   for more information on deployment statuses.
+
+2. ID - the deployment ID
+
```

Comment:
I have been using such formatting before (having an extra line) and seen it work fine. It may be the indentation of the paragraphs below. I'll experiment and fix it

Reply:
Looks like it was the indentation indeed. It works with 4 spaces:

![image](https://user-images.githubusercontent.com/889383/118972853-5006d480-b971-11eb-9052-07538b189510.png)

https://docs.cloudify.co/staging/RD-1529-documentation-for-Deployments-View/working_with/console/widgets/deploymentsview/

I think all the list issues are fixed, as far as I checked